python/cudf/cudf/core/column/lists.py

```diff
@@ -285,3 +285,56 @@ def take(self, lists_indices):
             raise
         else:
             return res
+
+    def sort_values(
+        self,
+        ascending=True,
+        inplace=False,
+        kind="quicksort",
+        na_position="last",
+        ignore_index=False,
+    ):
+        """
+        Sort each list by the values.
+
+        Sort the lists in ascending or descending order by some criterion.
+
+        Parameters
+        ----------
+        ascending : bool, default True
+            If True, sort values in ascending order, otherwise descending.
+        na_position : {'first', 'last'}, default 'last'
+            'first' puts nulls at the beginning, 'last' puts nulls at the end.
+        ignore_index : bool, default False
+            If True, the resulting axis will be labeled 0, 1, ..., n - 1.
+
+        Returns
+        -------
+        ListColumn with each list sorted
+
+        Notes
+        -----
+        Difference from pandas:
+          * Not supporting: `inplace`, `kind`
+
+        Examples
+        --------
+        >>> import cudf
+        >>> s = cudf.Series([[4, 2, None, 9], [8, 8, 2], [2, 1]])
```

Comment:
Can we add another example to our `Series.sort_values` which will show-case we can sort a series of listdtype too? Also, we might have to fix existing `Series.sort_values` examples where dtype of existing example seems to be cutoff, could you tackle that too along with your addition?

Reply:
Just to clarify, this is just to add an example in `Series.sort_values` docstring to show that we can sort the list elements by calling the list method - not to plumb the Series level API to the list method right? (if so, ) Should we use `See Also` tab for that?

Noted on fixing Series docstring cut-off issue.

Reply:
We can't sort list columns with `Series.sort_values` yet as far as I'm aware

Reply:
Ah, I'm sorry. I got mislead by seeing the example. In that case, let's start exposing our ListMethods in documentation and probably correct your example here to `s.list.sort_values` instead of `s.sort_values`

Reply:
Thanks for the catch. Noted on exposing list API.